### PR TITLE
Qt: Add custom pad configs

### PR DESCRIPTION
**Pull request by Megamouse**
Add a custom pad config for every game.

Configs will be saved per game in config/custom_input_configs

### CODE REVIEW

**Comment by Hasster1** [comment]
What happened with old pr?

File: rpcs3/rpcs3qt/game_list_frame.cpp [hunk 1]
```diff
@@ -711,7 +719,18 @@ void game_list_frame::ShowContextMenu(const QPoint &pos)
...
 			if (RemoveCustomConfiguration(currGame.serial, true))
 			{
-				ShowCustomConfigIcon(item, false);
+				ShowCustomConfigIcon(item, config::type::emu, false);
+			}
+		});
+	}
+	if (gameinfo->hasCustomPadConfig)
+	{
+		QAction* remove_custom_pad_config = remove_menu->addAction(tr("&Remove Custom Configuration"));
```
Comment (gidan80) [inline rpcs3/rpcs3qt/game_list_frame.cpp]: QAction* remove_custom_pad_config = remove_menu->addAction(tr("&Remove Custom Pad Configuration"));

Comment (Megamouse) [reply rpcs3/rpcs3qt/game_list_frame.cpp]: thx

**Comment by Megamouse** [comment]
I think I fixed all the problems @gidan80 found

**Comment by gidan80** [comment]
Thank you. Last suggestion/correction: while ingame, if you remove the custom config, the handler should fall back to the main one.

**Comment by Megamouse** [comment]
@gidan80 added that as well

**Comment by gidan80** [comment]
Well done. Thank you again for taking care. No bugs left i think, so you can merge it when you're ready.

**Pull request merged**